sentinel_connectors/sentinel_connectors/keyword_finder.py

```diff
@@ -3,17 +3,14 @@
 
 
 class KeywordFinder:
```

Comment:
I would simplify the naming and go for `AnyKeywordMatcher` with `match_any_keyword` and `AllKeywordsMatcher` with `match_all_keywords`.

Reply:
I have changed class names as you suggested. Methods names are changed to any_keyword_match and all_occuring_keywords. IMHO this is better because first returns bool and second one returns list of strings.